### DISCUSSION TITLE
Removing id3v1 tags

### DISCUSSION THREAD

**Discussion by JOJ0** [beetbox/beets#4219] [Q&A] [answered]
Hi, I might not have looked closely enough yet but is there a plugin existing that just gets rid of id3v1 tags entirely? beets is updating id3v2.4 as far as I understand and I guess I won't need id3v1 anymore for anything these days. Thanks for any hints in advance :-)

**Comment by sampsyo** [comment, marked as answer]
I'm not entirely sure it would work, but you might consider trying the `scrub` plugin.
